packages/aws-cdk/lib/commands/init/init.ts

```diff
@@ -500,7 +500,7 @@ interface Versions {
  * This has been built into the CLI at build time.
  */
 async function loadInitVersions(): Promise<Versions> {
-  const initVersionFile = path.join(__dirname, './init-templates/.init-version.json');
+  const initVersionFile = path.join(cliRootDir(), 'lib', 'init-templates', '.init-version.json');
```

Comment:
This was incorrect before and would have broken the toolkit-lib and cli-lib-alpha and integ-runner and cdk-cli-wrapper if they were supporting `init` command.